### DISCUSSION TITLE
Repository management

### DISCUSSION THREAD

**Discussion by dvanderfaeillie** [dodona-edu/feedback-and-support#7] [Questions - Vragen] [answered]
Because I want to implement a custom course I've added my own repository. I've noticed that I can't edit its details after adding the repo. I can't even delete my own repo. 

Since I forgot to use the git link (instead of the https link) my repo is useless. How should I proceed?

**Comment by chvp** [comment, marked as answer]
Send us an email at dodona@ugent.be. This will require manual action by an administrator to fix.

**Reply by dvanderfaeillie** [reply]
Thanks. Might be a feature to think about in future releases. (Easier own repo management)

**Reply by chvp** [reply]
There have been some issues created for it recently (#3210, #3211). Allowing to change the git link would be a hard issue though, since we would also need to change the remote URL of the underlying git clone, which would be a quite complicated operation.
